HiP-DataStore/Core/WriteModel/MediaIndex.cs

```diff
@@ -25,6 +26,24 @@ public bool IsPublishedAudio(int id)
                 info.Status == ContentStatus.Published &&
                 info.Type == MediaType.Audio;
         }
+        public MediaType? GetMediaType(int id)
+        {
+            lock (_lockObject)
```

Comment:
Locking is a good idea, but it only works if all calls to the _media dictionary use locking